### ISSUE TITLE
Suggestion: expose the epoch as a file in the rootfs

### ISSUE THREAD

**Issue by AkihiroSuda** [debuerreotype/debuerreotype#144]
The epoch is already exposed in `/etc/apt/sources.list` as comment lines, but hard to parse it robustly:
```console
root@b063b5cba10a:/# cat /etc/apt/sources.list
# deb http://snapshot.debian.org/archive/debian/20220912T000000Z bullseye main
deb http://deb.debian.org/debian bullseye main
# deb http://snapshot.debian.org/archive/debian-security/20220912T000000Z bullseye-security main
deb http://deb.debian.org/debian-security bullseye-security main
# deb http://snapshot.debian.org/archive/debian/20220912T000000Z bullseye-updates main
deb http://deb.debian.org/debian bullseye-updates main
```

Would it be possible to expose the epoch as a "plain" file too?
```console
root@b063b5cba10a:/# cat /.debuerreotype/epoch
1662940800

root@b063b5cba10a:/# cat /.debuerreotype/snapshot
20220912T000000Z
```

**Comment by tianon** [comment]
It's definitely possible, although probably a little bit unusual -- I'd love to build these images such that the `Created` metadata in the image itself is the epoch value (see https://github.com/docker-library/bashbrew/issues/51).

When I need to access the epoch in my own images, I usually use something like `stat --format '%Y' /etc/apt/sources.list` (or `stat --format '%Y' /etc/apt/sources.list.d/debian.sources` in the newer deb822-style images):

```console
$ docker run -it --rm debian:bullseye-slim
root@c8f1d8563492:/# stat --format '%Y' /etc/apt/sources.list
1662940800
```
